vividus-plugin-rest-api/src/main/java/org/vividus/archive/steps/ArchiveSteps.java

```diff
@@ -106,22 +112,40 @@ public void saveArchiveEntriesToVariables(DataWrapper archiveData, List<ArchiveV
     public void verifyArchiveContainsEntries(DataWrapper archiveData, List<NamedEntry> parameters)
     {
         Set<String> entryNames = ZipUtils.readZipEntryNamesFromBytes(archiveData.getBytes());
-
+        MutableBoolean isNeedToLoadAttachment = new MutableBoolean(false);
```

Comment:
What is the strategy here, attach archive when archive has entries and at least a single rule mismatched?

Reply:
yes, you are right